shipyard_blueprints/motherduck/shipyard_motherduck/errors.py

```diff
@@ -0,0 +1,23 @@
+from shipyard_templates import ExitCodeException
+
+EXIT_CODE_TEMP_TABLE_CREATION_ERROR = 101
+EXIT_CODE_TEMP_TABLE_INSERT_ERROR = 102
+EXIT_CODE_TEMP_TABLE_DROP_ERROR = 103
+
```

Comment:
can we name this file exceptions to be more consistent with the other blueprints.